pkg/jaeger/api.go

```diff
@@ -0,0 +1,25 @@
+package jaeger
+
+import (
+	"fmt"
+
+	"github.com/gorilla/mux"
+	jaegerQueryApp "github.com/jaegertracing/jaeger/cmd/query/app"
+	jaegerQueryService "github.com/jaegertracing/jaeger/cmd/query/app/querysvc"
+
+	"github.com/timescale/promscale/pkg/jaeger/query"
+	"github.com/timescale/promscale/pkg/pgxconn"
+	"github.com/timescale/promscale/pkg/telemetry"
+)
+
+func ExtendQueryAPIs(r *mux.Router, conn pgxconn.PgxConn, t telemetry.Engine) error {
+	reader, err := query.New(conn, t)
+	if err != nil {
+		return fmt.Errorf("error initializing tracing query: %w", err)
```

Comment:
Nit: `jaeger tracing query` since it's Jaeger specific

Reply:
No, over here, its the SQL module for tracing. Its not jaeger specific yet. It becomes Jaeger specific after this function has done its job.